### PR TITLE
Use frontend iframe for email signup

### PR DESCRIPTION
**Pull request by gidsg**
Use frontend iframe for email signup. This is required in order to do a secure signup.
screen shot 2016-08-30 at 15 19 41

CC @duarte @akash1810

### CODE REVIEW

File: src/js/lib/emailsignupURL.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function emailsignupURL(listId) {
+    return  window.location.origin === 'https://www.theguardian.com' ? 'https://www.theguardian.com/email/form/plaindark/' + listId : 'https://m.code.dev-theguardian.com/email/form/plaindark/' + listId;
```
Comment (akash1810) [inline src/js/lib/emailsignupURL.js]: we can use [template strings](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) here - ❤️ ES6

File: src/js/main.js [hunk 2]
```diff
@@ -21,6 +22,7 @@ export function init(el, context, config) {
             builder.querySelector('.docs__poster--play-button').setAttribute('data-duration', duration);
         });
 
+        pimpYouTubePlayer(youTubeId, builder.querySelector('#playerWrapper'), '100%', '100%');
```
Comment (akash1810) [inline src/js/main.js]: we're calling `pimpYouTubePlayer` on line 64 already, don't think this needed?

File: src/js/text/main.html [hunk 3]
```diff
@@ -82,11 +82,8 @@
             We release a new documentary every month.<br>
             Would you like a reminder with new releases?
           </p>
-          <form class='email-form__form'>
-            <label for='email' class='email-address'>Email address</label>
-            <input type='email' id='email' placeholder="Email address">
-            <input type='submit' id='remind-me' value='Remind me'>
-          </form>
+
+            <iframe height="131px" data-form-success-headline="Thank you" data-form-success-desc="We will send you reminders about forthcoming documentaries." scrolling="no" seamless="" frameborder="0" class="iframed--overflow-hidden email-sub__iframe email-sub__iframe--end-article js-email-sub__iframe js-email-sub__iframe--article"></iframe>
```
Comment (akash1810) [inline src/js/text/main.html]: what does `seemless=""` do?

File: src/js/main.js [hunk 4]
```diff
@@ -62,6 +64,10 @@ export function init(el, context, config) {
 
 
         pimpYouTubePlayer(youTubeId, builder.querySelector('#playerWrapper'), '100%', '100%', builder);
+
+        const emailIframe = builder.querySelector('.js-email-sub__iframe');
+        emailIframe.setAttribute('src', emailsignupURL(37));
```
Comment (akash1810) [inline src/js/main.js]: can the `37` go in the config?

File: src/js/text/main.html (hunk 3 above)
Comment (gidsg) [inline src/js/text/main.html]: Looks like this got depreciated so will remove: http://caniuse.com/#feat=iframe-seamless

**Comment by akash1810** [comment]
👍

File: src/js/boot.js.tpl [hunk 5]
```diff
@@ -19,7 +19,8 @@ define([], function() {
                 'assetPath': '<%= assetPath %>',
                 'sheetName': 'gun-nation',
                 'sheetChapter': 'gun-nation-chapters',
-                'sheetId': '1NcSvccw77rHAncarwfeq7RGZF7yez9mP1Icf3oLMA7g'
+                'sheetId': '1NcSvccw77rHAncarwfeq7RGZF7yez9mP1Icf3oLMA7g',
+                'emailListId': '3745'
```
Comment (akash1810) [inline src/js/boot.js.tpl]: can this be an int rather than a string?

**Pull request merged**